packages/@okta/vuepress-site/docs/concepts/multi-tenancy/index.md

```diff
@@ -307,10 +317,11 @@ Reasons why organizations choose this configuration include:
 isolated when placed into separate orgs. This isolation becomes relevant
 when user data is required to reside in a specific region (for
 example, government regulations).
-* **Strong delegated admin support:** Although there's delegated admin
+* **Strong delegated admin support:**
+ <!-- Although there's delegated admin
 support in [okta-dac](#the-okta-dac-project), the number of delegated
-responsibilities that are supported is limited. Splitting customers and
-partners into separate orgs offer the maximum level of delegated
+responsibilities that are supported is limited.-->
+Splitting customers and partners into separate orgs offer the maximum level of delegated
```

Comment:
```suggestion
Splitting customers and partners into separate orgs offers the maximum level of delegated
```

Reply:
Treating the phrase "Splitting customers and partners into separate orgs" as singular, so change the verb to "offers"